types/properties-reader/index.d.ts

```diff
@@ -18,8 +18,8 @@ declare namespace PropertiesReader {
         each<T>(iterator: (this: T, key: string, value: Value) => void, scope: T): Reader;
         getAllProperties(): { [key: string]: Value; };
         clone(): Reader;
-        save(destFile: string): Promise<any>;
-        getByRoot(root: any): {};
+        save(destFile: string, onComplete?: Function ): Promise<any>;
```

Comment:
Since the documentation specifies it, we can use a more specific callback type here, something like `(err: any, data: string) => void` instead of `Function`.

I can see now in the source that the returned result is actually the contents of the saved file, so we can also specify the return type as `Promise<string>`.